core/commands/commit.py

```diff
@@ -64,9 +64,12 @@ def run_async(self, repo_path=None, include_unstaged=False, amend=False):
         else:
             view.set_syntax_file("Packages/GitSavvy/syntax/make_commit.tmLanguage")
 
+        commit_on_close = savvy_settings.get("commit_on_close")
+        view.settings().set("git_savvy.commit_on_close", commit_on_close)
```

Comment:
It *is used* in the [this commit](https://github.com/asfaltboy/GitSavvy/commit/4e20d1639dbb94ae1b3c39ffce001ceadb192638) to check toggle super/ctrl+enter off in commit view when using `commit_on_close`.